integration_tests/dbt_project.yml

```diff
@@ -33,6 +33,8 @@ vars:
     workday_worker_leave_status_identifier: "workday_worker_leave_status_data"
     workday_worker_position_organization_history_identifier: "workday_worker_position_organization_history_data"
 
+    employee_history_enabled: true
```

Comment:
Good call-out. I'd set this to true for regenerating docs since otherwise the new models will not appear. But I will  use this as a reminder to set to false before merging.